django/db/backends/mysql/features.py

```diff
@@ -30,6 +30,7 @@ class DatabaseFeatures(BaseDatabaseFeatures):
     supports_select_intersection = False
     supports_select_difference = False
     supports_slicing_ordering_in_compound = True
+    supports_index_on_textfield = False
```

Comment:
I would rather add `supports_index_on_textfield = False` in `backends/base/features.py` and single exception (`supports_index_on_textfield = True`) on postgreSQL (`backends/potgresql/features.py`).